test/REQUIRE

```diff
@@ -1,2 +1,2 @@
-DataStructures
+DataStructures 0.5.2
```

Comment:
if the package were 0.6-only, or there were a better way to say "apply this minimum package version for this version of julia" then it would make sense, but yeah for now maybe shouldn't restrict on 0.5 if not needed